src/sentry/notifications/notifications/activity/release.py

```diff
@@ -87,8 +90,7 @@ def get_projects(self, recipient: RpcActor) -> set[Project]:
             return set()
 
         if recipient.actor_type == ActorType.USER:
-            if recipient.is_superuser or self.organization.flags.allow_joinleave:
```

Comment:
removing is_superuser is a material change. Is that not available?

Reply:
Not on RpcActor. Superuser attributes aren't on the actor, so we'd need to do RPC calls for each and every actor read to hydrate this attribute that is only used here. We talked about this during office hours and decided that removing this attribute from actors was simpler. This logic also lacks tests and felt like a bonus inclusion from the original pull request.